packages/fast-development-site-react/src/components/site/dev-tools.tsx

```diff
@@ -10,7 +10,11 @@ export enum FrameworkEnum {
     angular = "angular"
 }
 
-export type tabType = "Code" | "Properties" | "Schema";
+export enum TabTypeEnum {
```

Comment:
I know that we have it on the "FrameworkEnum" above, but I think that's actually a mistake. Adding enum to a typed "enum" seems totally redundant to me. I'm sure you were following the existing pattern, but I don't know that it's the pattern we want.

Reply:
Sure, I can remove the word Enum - I can see how it is similar to saying  "ATM machine" :)